frontend/src/components/GrantsIdentifier.tsx

```diff
@@ -0,0 +1,106 @@
+import { Trans, useTranslation } from "next-i18next";
+import Image from "next/image";
+import {
+  Identifier,
+  IdentifierGov,
+  IdentifierIdentity,
+  IdentifierLink,
+  IdentifierLinkItem,
+  IdentifierLinks,
+  IdentifierLogo,
+  IdentifierLogos,
+  IdentifierMasthead,
+} from "@trussworks/react-uswds";
+
+import logo from "../../public/img/logo-white-lg.webp";
+
+const GrantsIdentifier = () => {
+  const { t } = useTranslation("common", {
+    keyPrefix: "Identifier",
+  });
+
+  const logoImage = () => {
+    return (
+      <Image
+        alt={t("logo_alt")}
+        src={logo}
+        height={48}
+        width={48}
+        style={{
+          width: "48px",
+          height: "48px",
+        }}
+      />
+    );
+  };
+
+  return (
+    <Identifier data-testid="identifier">
+      <IdentifierMasthead aria-label="Agency identifier">
+        <IdentifierLogos>
+          <IdentifierLogo href="#">{logoImage()}</IdentifierLogo>
+        </IdentifierLogos>
+        <IdentifierIdentity domain="HHS.gov">
+          <Trans
+            t={t}
+            i18nKey="identity"
+            components={{
+              hhsLink: <a href="https://www.hhs.gov" />,
+            }}
+          />
+        </IdentifierIdentity>
+      </IdentifierMasthead>
+      <IdentifierLinks navProps={{ "aria-label": "Important links" }}>
```

Comment:
Just a thought/question (please take w/ a grain of salt): Would it be possible to loop thru these links and code the element only once? Haven't thought thru exactly how that'd work… it'd mean moving the URIs to a separate file with their texts, a lookup table (ew), or something. Anyway, other NextJS projects Nave has kept all external links in an object (`routes.external.[agency]`) in the routes file. Maybe we don't need that much organization yet, or need to over-optimize, and hardcoding here is fine. 🤷

Reply:
Is this what you had in mind?

Reply:
Ooh! Slick. Yeah. I like that the markup is just in there once, and it'll be easy if we need to add/edit/remove links.